apps/studio/components/interfaces/Integrations/Vault/Secrets/SecretsManagement.tsx

```diff
@@ -167,10 +165,6 @@ export const SecretsManagement = () => {
         </div>
       </div>
 
-      <EditSecretModal
-        selectedSecret={selectedSecretToEdit}
-        onClose={() => setSelectedSecretToEdit(undefined)}
-      />
       <DeleteSecretModal
         selectedSecret={selectedSecretToRemove}
         onClose={() => setSelectedSecretToRemove(undefined)}
```

Comment:
both of these get moved into `SecretRow` because it doesn't really make sense to have this awkward sibling relationship where the dialog trigger is nested, causing state management in the parent. I think it's the kind of wonky coupling of components that we'd want to avoid